packages/ui/src/components/vuestic-components/va-app-bar/VaAppBar.demo.vue

```diff
@@ -4,7 +4,7 @@
       <div class="frame">
         <div class="display">
           <va-app-bar>
-            <AppBarTestContent />
```

Comment:
any reason to have gray everywhere?

Reply:
I pass it a gray color to make the content stand out from the app bar, otherwise it won't be visible. I am not passing it only where the color prop of the app-bar itself needs to be checked